openshift/e2e-common.sh

```diff
@@ -495,6 +496,30 @@ function run_e2e_tests(){
     --skip-cleanup-on-fail \
     --resolvabledomain || failed=1
 
+
+  # Verify that the right sc is set by default and seccompProfile is injected on OCP >= 4.11.
+  go_test_e2e -timeout=3m ./test/e2e/securedefaults -run "^(TestSecureDefaults)$" \
+    --kubeconfig "$KUBECONFIG" \
+    --imagetemplate "$TEST_IMAGE_TEMPLATE" \
+    --enable-alpha \
+    --https \
+    --skip-cleanup-on-fail \
+    --resolvabledomain || failed=1
+
+  # Allow to use any seccompProfile for non default cases,
+  # for more check https://docs.openshift.com/container-platform/4.12/authentication/managing-security-context-constraints.html
+  oc adm policy add-scc-to-user privileged -z default -n serving-tests
+
+  # Verify that non secure settings are allowed, although not-recommended.
+  go_test_e2e -timeout=3m ./test/e2e/securedefaults -run "^(TestUnsafePermitted)$" \
```

Comment:
It might be better to use some longer timeout, e.g. 10 minutes. Some "poll" timeouts inside the Golang tests are longer (10 minutes).